diff --git a/pythonmstk/pythonmstk.py b/pythonmstk/pythonmstk.py
index a22659c..718ec52 100644
--- a/pythonmstk/pythonmstk.py
+++ b/pythonmstk/pythonmstk.py
@@ -71,7 +71,7 @@ class MstkServer:
       self.activity_logfile = ('/var/log/%s.activity.log' % daemon)
       open(self.activity_logfile, 'a').close()
       # RYAN
-      self.debug_message(self.log_level,0, "test")
+      self.debug_message(self.log_level,0, "Restarted")
       self.slack.chat_postMessage(channel=str(self.slack_channel),text="Void's grip released. Initiating connection to Slack. Resuming sentinel duty. New arrivals shall be announced once more.") # Fixed hardcoded slack channel

    def debug_message(self,current_log_level, message_level, message):
@@ -158,17 +158,19 @@ class MstkServer:
          if todays_entries == 0:
             logline = '{} {} {} {} {}{}'.format(entryname,badge1,len(date_integers)+1,badge2,streak,badge_message) # Updated doorbot msg post format - 1/22/22 DWC (And total Days Bug, XlogicX)
             self.slack.chat_postMessage(channel=str(slack_channel),text=logline) # Fixed hardcoded slack channel
-            try:
-               requests.post(
-                  URL,
-                  json={
-                     "timestamp": datetime.now().isoformat(),
-                     "name": entryname,
-                     "access_granted": True,
-                  },
-               )
-            except Exception as e:
-               pass
+
+         try:
+            r = requests.post(
+               DOORBOT2_URL,
+               json={
+                  "timestamp": datetime.now().isoformat(),
+                  "name": entryname,
+                  "access_granted": True,
+               },
+            )
+            r.raise_for_status()
+         except Exception as e:
+            self.debug_message(self.log_level, 3, f"error sending access log to doorbot2: {e}")

          currenttime  = time.strftime("%H:%M:%S") # XlogicX debugging
          debugfile = open("/var/log/doorbotdebug.txt", "a"); # XlogicX debugging
